daemon/buildinfo.c

```diff
@@ -266,9 +266,8 @@ void print_build_info(void) {
     printf("    dbengine:                   %s\n", FEAT_YES_NO(FEAT_DBENGINE));
     printf("    Native HTTPS:               %s\n", FEAT_YES_NO(FEAT_NATIVE_HTTPS));
     printf("    Netdata Cloud:              %s %s\n", FEAT_YES_NO(FEAT_CLOUD), FEAT_CLOUD_MSG);
-    printf("    ACLK Next Generation:       %s\n", FEAT_YES_NO(FEAT_CLOUD));
-    printf("    ACLK-NG New Cloud Protocol: %s\n", FEAT_YES_NO(1));
-    printf("    ACLK Legacy:                %s\n", FEAT_YES_NO(0));
+    printf("    ACLK:                       %s\n", FEAT_YES_NO(FEAT_CLOUD));
+    printf("    ACLK New Cloud Protocol:    %s\n", FEAT_YES_NO(1));
```

Comment:
Hi @underhood !

Should we maybe simplify things and remove `ACLK New Cloud Protocol:` ? I guess we can't have ACLK at all now if e.g. protobuf can not be built/missing ?

Or even just keep `Netdata Cloud` only, i.e. either there is capability to connect to the cloud or there's not ?

Reply:
true, thx